onnxruntime/test/contrib_ops/matmul_integer_to_float_test.cc

```diff
@@ -221,5 +243,68 @@ TEST(MatMulIntegerToFloat, MatMulInteger_With_ZeroPoint) {
   test_case({15, 14, 13}, {15, 13, 27}, {15, 1, 27});
 }
 
+TEST(MatMulIntegerToFloat, MatMulInteger_With_ZeroPoint_FP16) {
+  auto test_case = [&](const std::vector<int64_t>& input_shape,
+                       const std::vector<int64_t>& weights_shape,
+                       const std::vector<int64_t>& b_scale_zp_shape) {
+    auto build_test_case = [&](ModelTestBuilder& builder) {
+      auto* input_arg = builder.MakeInput<uint8_t>(input_shape,
+                                                   std::numeric_limits<uint8_t>::min(),
+                                                   std::numeric_limits<uint8_t>::max());
+      auto* output_arg = builder.MakeOutput();
+      auto* weight = builder.MakeInitializer<int8_t>(weights_shape,
+                                                     std::numeric_limits<int8_t>::min() / 2,
+                                                     std::numeric_limits<int8_t>::max() / 2);
+
+      // add MatMulInteger
+      auto* matmul_integer_output = builder.MakeIntermediate();
+      auto* A_zp_arg = builder.MakeInput<uint8_t>({1},
+                                                  std::numeric_limits<uint8_t>::min(),
+                                                  std::numeric_limits<uint8_t>::max());
+      auto* B_zp_arg = builder.MakeInput<int8_t>(b_scale_zp_shape,
+                                                 std::numeric_limits<int8_t>::min() / 2,
+                                                 std::numeric_limits<int8_t>::max() / 2);
```

Comment:
>std::numeric_limits<int8_t>::max() / 2 [](http://example.com/codeflow?start=49&length=38)

Just double checking, do you mean for this to be 63? (or 127?)

Reply:
This part is picked as in from [ MatMulInteger_With_ZeroPoint](https://github.com/microsoft/onnxruntime/pull/16871/files/f3030db2c64f2e779597b904628762f5662e26c6#diff-bcf468a0845e26141b51342a82452466ba35d3d650e9599d17d08b9e519305ecR192-R203)  written by contrib op dev, I believe it is to limit overflow